### PR TITLE
COVID19 dataset tab and portal to covid19-3.0.0

### PR DESCRIPTION
**Pull request by paulineribeyre**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

File: chicagoland.pandemicresponsecommons.org/portal/gitops.json [hunk 1]
```diff
@@ -125,68 +125,107 @@
...
+          "data_description",
+          "code"
+        ]
       },
-      "source": {
-        "chartType": "pie",
-        "title": "Sources"
+      "buttons": [],
+      "guppyConfig": {
+        "dataType": "dataset",
```
Comment (mfshao) [inline chicagoland.pandemicresponsecommons.org/portal/gitops.json:152]: is this index ready? i don't see it in the manifest

Comment (paulineribeyre) [reply chicagoland.pandemicresponsecommons.org/portal/gitops.json:152]: https://github.com/uc-cdis/cdis-manifest/blob/master/chicagoland.pandemicresponsecommons.org/manifest.json#L182

Comment (mfshao) [reply chicagoland.pandemicresponsecommons.org/portal/gitops.json:152]: uh... missed that one. thanks

**Pull request merged**